_pytest/pytester.py

```diff
@@ -1077,6 +1077,23 @@ def _getlines(self, lines2):
         return lines2
 
     def fnmatch_lines_random(self, lines2):
+        """Check lines exist in the output using ``fnmatch.fnmatch``, in any order.
+
+        The argument is a list of lines which have to occur in the
+        output, in any order.
+        """
+        self._match_lines_random(lines2, fnmatch)
+
+    def re_match_lines_random(self, lines2):
+        """Check lines exist in the output.
+
+        The argument is a list of lines which have to occur in the
+        output, in any order.  Each line can contain glob whildcards.
```

Comment:
whildcards -> wildcards - but it looks like this actually takes a regex pattern, and not globs?

Reply:
done, thanks